bouncer/tests/boost.ts

```diff
@@ -1,118 +1,121 @@
+  const extrinsicResult = await cf.stepToExtrinsicIncluded((api) =>
+    api.tx.lendingPools.stopBoosting(shortChainFromAsset(asset).toUpperCase(), boostTier),
   );
-  if (!extrinsicResult?.dispatchError) {
-    logger.debug('waiting for stop boosting event');
-    return observeStoppedBoosting;
+
+  if (extrinsicResult.ok) {
+    cf.info('waiting for stop boosting event');
+    return cf.expectEvent(
+      'LendingPools.StoppedBoosting',
+      lendingPoolsStoppedBoosting.refine(
+        (event) =>
+          event.boosterId === cf.requirements.account.keypair.address &&
+          event.boostPool.asset === asset &&
+          event.boostPool.tier === boostTier,
+      ),
+    );
   }
-  logger.debug('Already stopped boosting');
+
+  cf.info(`Already stopped boosting (${extrinsicResult.error})`);
   return undefined;
 }
 
 /// Adds existing funds to the boost pool of the given tier and returns the BoostFundsAdded event.
 export async function addBoostFunds(
-  logger: Logger,
+  cf: ChainflipIO<WithLpAccount>,
   asset: Asset,
   boostTier: number,
   amount: number,
-  lpUri = '//LP_BOOST',
-): Promise<Event> {
-  await using chainflip = await getChainflipApi();
-  const lp = createStateChainKeypair(lpUri);
-  const extrinsicSubmitter = new ChainflipExtrinsicSubmitter(lp, lpMutex.for(lpUri));
-
+): Promise<z.infer<typeof lendingPoolsBoostFundsAdded>> {
   assert(boostTier > 0, 'Boost tier must be greater than 0');
 
-  const observeBoostFundsAdded = observeEvent(logger, `lendingPools:BoostFundsAdded`, {
-    test: (event) =>
-      event.data.boosterId === lp.address &&
-      event.data.boostPool.asset === asset &&
-      event.data.boostPool.tier === boostTier.toString(),
-  });
-
   // Add funds to the boost pool
-  logger.debug(`Adding boost funds of ${amount} ${asset} at ${boostTier}bps`);
-  await extrinsicSubmitter.submit(
-    chainflip.tx.lendingPools.addBoostFunds(
+  cf.debug(`Adding boost funds of ${amount} ${asset} at ${boostTier}bps`);
+  await cf.stepToExtrinsicIncluded((api) =>
+    api.tx.lendingPools.addBoostFunds(
       shortChainFromAsset(asset).toUpperCase(),
       amountToFineAmount(amount.toString(), assetDecimals(asset)),
       boostTier,
     ),
   );
 
-  return observeBoostFundsAdded.event;
+  const result = await cf.expectEvent(
+    'LendingPools.BoostFundsAdded',
+    lendingPoolsBoostFundsAdded.refine(
+      (event) =>
+        event.boosterId === cf.requirements.account.keypair.address &&
+        event.boostPool.asset === asset &&
+        event.boostPool.tier === boostTier,
+    ),
+  );
+
+  return result;
 }
 
 /// Adds boost funds to the boost pool and does a swap with boosting enabled, then stops boosting and checks the fees collected are correct.
 async function testBoostingForAsset(
   asset: Asset,
   boostFee: number,
-  lpUri: string,
+  lpUri: `//${string}`,
   amount: number,
   testContext: TestContext,
 ) {
+  const cf: ChainflipIO<WithLpAccount> = new ChainflipIO(testContext.logger, {
```

Comment:
```suggestion
  const cf: ChainflipIO<WithLpAccount> = new ChainflipIO(testContext.logger.child({ boostAsset: asset, boostFee }), {
```
Then we can remove all uses of `logger' and just use the `cf` logger. At the moment its a mix of both loggers.

Reply:
Good catch, yeah I did that.